ktor-client/ktor-client-webrtc/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:OptIn(ExperimentalWasmDsl::class)
+
+import org.jetbrains.kotlin.gradle.ExperimentalWasmDsl
+
+description = "Ktor WebRTC client"
+
+plugins {
+    id("com.android.library")
+    id("kotlinx-serialization")
+    id("ktorbuild.project.library")
+}
+
+kotlin {
+    jvmToolchain(17)
+
+    androidTarget {
+        publishLibraryVariants("release")
```

Comment:
This could be moved to the place where we declare `androidTarget` the first time, as it will be needed for any android module.